addons/static/templates/policy_cache_transformer_persona.json

```diff
@@ -394,9 +394,15 @@
       "resources": [
         "entity:{entity}",
         "entity-type:DataDomain",
-        "entity-classification:*"
+        "entity-classification:*",
+        "classification:*"
       ],
-      "actions": ["entity-update"]
+      "actions": [
+        "entity-update",
+        "entity-add-classification",
+        "entity-update-classification",
+        "entity-remove-classification"
```

Comment:
We have never clubbed entity permissions & tag permission like this. I hope you have tested & this works for both entity-update & tag permissions

Reply:
Yes